### ISSUE TITLE
Add the variable to make a tree for b-form-select

### ISSUE THREAD

**Issue by ab2d3e8** [conventional-changelog/standard-version#330]
Is it possible to use '<b-form-select />' for a '<optgroup >' from variable:

 options: [
                    { value: 'A', text: 'Option A' },
                    { value: 'B', text: 'Option B' ,
                      children: [
                                  { value: 'BA', text: 'Option BA' },
                                  { value: 'BB', text: 'Option BB' }
                                ]
                    }
                  ],

**Comment by bcoe** [comment]
@ab2d3e8 I'm not following what you're trying to do, are you sure this is the right library?

**Comment by ab2d3e8** [comment]
You are right, this is not the correct library, this is the library bootstrap
